lib/options.py

```diff
@@ -252,6 +260,11 @@ def __init__(self):
                 action="store_true",
                 default=False,
                 help="""Disable schema upgrade on testing with snapshots.""")
+        parser.add_argument(
+                "--allocator",
+                dest="allocator",
+                default=env_string('TEST_RUN_MEMTX_ALLOCATOR', "small"),
```

Comment:
Regarding prefixing the environment variable name with `TEST_RUN_` or not.

1. There are `TEST_RUN_TESTS` and `TEST_RUN_JOBS` variables, which configures test-run itself. However more recent `TEST_TIMEOUT` and `NO_OUTPUT_TIMEOUT` were added without the prefix. To be honest, I just don't find any gains and decided to get rid of the prefix to make it more convenient in use.
2. There are several other `TEST_RUN_*` variables, which are set by test-run itself for child processes (such as `TEST_RUN_WORKER_ID`).
3. There is `REPLICATION_SYNC_TIMEOUT` environment variable (and `--replication-sync-timeout` parameter) that reflects the box.cfg option name. It have no `TEST_RUN_` prefix.

Our case is quite similar to the third one. So, if there are no complains, I would follow this naming.

Reply:
Now it is `MEMTX_ALLOCATOR` without the prefix. I think it is good choice.